src/github/nag.rs

```diff
@@ -744,11 +747,14 @@ fn process_poll
     use domain::schema::poll_response_request;
     let conn = &*DB_POOL.get()?;
 
+    let tmp_teams;
```

Comment:
Move temporary inside `if`?

Reply:
I can't do that, the tmp is to keep the owned list of teams alive until the end of the function: if I put it inside the `if` it won't compile.

Reply:
Ah; 👍